turtlebot2_drivers/launch/turtlebot_carto_2d.py

```diff
@@ -35,21 +36,29 @@ def launch(launch_descriptor, argv):
         exit_handler=restart_exit_handler,
     )
     ld.add_process(
-        # The XYZ numbers for base_link -> camera_rgb_frame are taken from the turtlebot URDF
-        # in https://github.com/turtlebot/turtlebot/blob/kinetic/turtlebot_description/urdf/sensors/astra.urdf.xacro
+        # The XYZ/Quat numbers for base_link -> camera_rgb_frame are taken from the
+        # turtlebot URDF in
+        # https://github.com/turtlebot/turtlebot/blob/kinetic/turtlebot_description/urdf/sensors/astra.urdf.xacro
```

Comment:
please pin down specific commits when referring to files from another repository (same below)